src/SmartComponents/Applications/Applications.js

```diff
@@ -1,20 +1,55 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { PageHeader, PageHeaderTitle } from '@redhat-cloud-services/frontend-components';
+import { connect } from 'react-redux';
+import { Skeleton, PageHeader, PageHeaderTitle, Main } from '@redhat-cloud-services/frontend-components';
+import FormRender from '@data-driven-forms/react-form-renderer';
+import { formFieldsMapper, layoutMapper } from '@data-driven-forms/pf4-component-mapper';
+import registryDecorator from '@redhat-cloud-services/frontend-components-utilities/files/Registry';
+import { register } from '../../store';
+import reducers  from '../../store/reduces.js';
+import { getSchema } from '../../actions';
 
+const localStorageKey = (appName, user) => `@@settings-${appName}-${user}`;
+
+@registryDecorator()
 class Applications extends Component {
+    constructor(props) {
+        super(props);
+        register(reducers);
+        props.getSchema(props.match.params.id);
+        this.state = {
+            appName: this.props.match.params.id
+        };
+    }
+
+    componentDidMount() {
+        insights.chrome.auth.getUser().then(auth => ({ user: auth.identity.user.username })).then(user => this.setState(user));
```

Comment:
This doesn't have to be here, instead you can call this in mocked set api call and use it there.